### ISSUE TITLE
Copy/Download Server Files (Change File Manager)

### ISSUE THREAD

**Issue by not-3than** [Minehut/Meta#609]
### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information 

**Platform**

Website/In-Game
**Describe the feature**

Copy Server Button OR File Downloading
**How the feature is useful**

I'd like a copy server button so I can test new features on my server before adding them to the main one. I have struggled to create a copy of my own server as a dev server to just test stuff on, it would be nice if we could copy files.
If not that, then allow players to download files (except for certain plugin jars which are restricted to Minehut). It would be easier to download files and folders directly to my PC in case I wanted to make a dev server on localhost or something like that.
Basically: Just add more freedom to the files system! *(I know this may be a longshot, but maybe an FTP?)*
---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by quick007** [comment]
Duplicate of #237

**Comment by not-3than** [comment]
> Duplicate of #237

Didn't bother to read the thing before saying that...

**Comment by quick007** [comment]
Yea, you would copy all ur files and paste them on another server

**Comment by Sniped** [comment]
Giving this a read, this issue is suggesting directly downloading server files as well as copying them cross-server, this makes this issue a duplicate of both #11 and #237. Since you also suggest that Minehut add the ability to connect to servers via FTP, that would also make this issue a duplicate of #538.
